docs/FUZZING.md

```diff
@@ -0,0 +1,77 @@
+# Fuzzing
+
+Fuzz testing is an automated software testing method that injects invalid, 
+malformed, or unexpected inputs to reveal defects and vulnerabilities. A fuzzing 
+tool monitors the system for exceptions like crashes, information leakage, or 
+errors, helping developers identify and fix bugs and security loopholes.
+
+## Current state of fuzzing in liboqs
+- [ ] kem 
+  - [ ] bike.md
+  - [ ] classic_mceliece.md
+  - [ ] frodokem.md
+  - [ ] hqc.md
+  - [ ] kyber.md
+  - [ ] ml_kem.md
+  - [ ] ntruprime.md
+- [ ] sig
+  - [x] dilithium.md
+  - [ ] falcon.md
+  - [ ] mayo.md
+  - [ ] ml_dsa.md
+  - [ ] sphincs.md
+- [ ] sig_stfl
+  - [ ] lms.md
+  - [ ] sig_stfl.md
+  - [ ] xmss.md
+
+## Building and running fuzz tests
+
+Building fuzz tests is very similar to building normally with some optional
+steps to target different types of bugs. The most basic ways to build the
+fuzz tests is as follows;
+
+```bash
+$ mkdir build && cd build
```

Comment:
Another minor point: the dollar signs look nice but make it annoying to copy/paste shell commands (especially multiline blocks).

Reply:
Done